common/lib/xmodule/xmodule/modulestore/xml_importer.py

```diff
@@ -173,9 +174,9 @@ def import_static_file(self, full_file_path, base_dir):
         try:
             self.static_content_store.save(content)
         except Exception as err:
-            log.exception(u'Error importing {0}, error={1}'.format(
-                file_subpath, err
-            ))
+            msg = "Error importing {0}, error={1}".format(file_subpath, err)
+            log.exception(msg)
+            set_custom_attribute('course_import_failure', msg)
```

Comment:
One potential improvement would be to do something like the following:
```
set_custom_attribute('course_import_failure', '[XXX]: {}'.format(msg))
```
You would replace 'XXX' with something that differentiates the different calls, like the method name, etc.